Prox/Prox/PlaceCarousel/PlaceCarouselCollectionViewCell.swift

```diff
@@ -120,13 +122,11 @@ class PlaceCarouselCollectionViewCell: UICollectionViewCell {
     override var isSelected: Bool {
         didSet {
             if isSelected {
-                shadowView.layer.shadowOffset = CGSize(width: 2, height: 2)
-                shadowView.layer.shadowOpacity = 0.5
-                shadowView.layer.shadowRadius = 2
+                shadowView.layer.shadowOffset = CGSize(width: 0, height: 2)
+                shadowView.layer.shadowRadius = 5
             } else {
-                shadowView.layer.shadowOffset = CGSize(width: 1, height: 1)
-                shadowView.layer.shadowOpacity = 0.25
-                shadowView.layer.shadowRadius = 1
+                shadowView.layer.shadowOffset = CGSize(width: 0, height: 0.5)
+                shadowView.layer.shadowRadius = 2
```

Comment:
Can we make some constants for the shadowOffset and shadowRadius values and use those instead? It will make things easier if we have to tweak further